packages/codemods/react-native/74/migrate-to-fire-interval-seconds/README.md

```diff
@@ -1 +1,19 @@
-Migrate from `repeatInterval` to `fireIntervalSeconds`.
\ No newline at end of file
+This codemod migrates `PushNotificationIOS.scheduleLocalNotification` deprecated `repeatInterval` to `fireIntervalSeconds`.
```

Comment:
Ideally each README should have the following structure: 
```
# CodemodName
## Description
Short description paragrapth
/n
Detailed description paragraph
## Examples
### Before: 
### After:   
```
If you follow this sctructure, codemod registry will be able to pick short description and render it in the codemod list.

Reply:
Example: https://github.com/codemod-com/codemod/blob/afee0431a8ebe87e68a6e8356a115ee6203ae288/packages/codemods/react/19/remove-context-provider/README.md